cmd/textiled/main.go

```diff
@@ -53,10 +53,6 @@ var (
 			Key:      "addr.gateway.url",
 			DefValue: "http://127.0.0.1:8006",
 		},
-		"addrGatewayBucketDomain": {
```

Comment:
I can't remember why I added this... the DNS domain setting can be used instead.